File: PageObjectPython/PageObjects/AutomationPracticeStore/StorePage.py
from PageObjects.AutomationPracticeStore.Objects.Footer import Footer
from PageObjects.AutomationPracticeStore.Objects.Header import Header
from PageObjects.BasePageObject import BasePageObject


class StorePage(BasePageObject):

    def get_header(self):
        return Header()

    def get_footer(self):
        return Footer()

File: PageObjectJava/src/test/java/PageObjects/AutomationPracticeStore/StorePage.java
package PageObjects.AutomationPracticeStore;

import Base.BasePageObject;
import PageObjects.AutomationPracticeStore.Objects.Footer;
import PageObjects.AutomationPracticeStore.Objects.Header;

public class StorePage extends BasePageObject {

    public Header getHeader(){
        return new Header();
    }

    public Footer getFooter(){
        return new Footer();
    }

}
